infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/BrokerEnvironmentFactory.java

```diff
@@ -87,13 +91,18 @@ public BrokerEnvironmentFactory(
       String artifactsBrokerImage,
       String metadataBrokerImage,
       String pluginRegistryUrl,
+      String pluginRegistryInternalUrl,
       CertificateProvisioner certProvisioner) {
     this.cheWebsocketEndpoint = cheWebsocketEndpoint;
     this.brokerPullPolicy = brokerPullPolicy;
     this.authEnableEnvVarProvider = authEnableEnvVarProvider;
     this.machineTokenEnvVarProvider = machineTokenEnvVarProvider;
     this.artifactsBrokerImage = artifactsBrokerImage;
     this.metadataBrokerImage = metadataBrokerImage;
+    if (!isNullOrEmpty(pluginRegistryInternalUrl)) {
+      pluginRegistryUrl = pluginRegistryInternalUrl;
+    }
+    LOGGER.info("Plugin registry url: {}", pluginRegistryUrl);
```

Comment:
It is better to print on a row below
`LOGGER.info("Plugin registry url: {}", this.pluginRegistryUrl);`